app/Http/Controllers/Auth/ResetPasswordController.php

```diff
@@ -83,7 +83,7 @@ protected function rules()
             'email' => 'required|email',
             'password' => [
                 'required',
-                'min:8',
+                'min:9',
```

Comment:
If you go to `PasswordFormatRule.php` and look for the [regex](https://github.com/GCTC-NTGC/TalentCloud/blob/95a834e26598b42729d2147a2379717aac2c2ed4/app/Services/Validation/Rules/PasswordFormatRule.php#L14) under `passwordPattern`. It says that it accepts a minimum of 8 characters. I'm not sure if the `min` validation rule here takes priority over it, but maybe we should change the regex just incase it doesn't.

Reply:
@yonikid15 I'm not sure one takes priority over the other, I think both rules are applied, based on testing.